communication.md

```diff
@@ -0,0 +1,75 @@
+# Communicating in Gizra
+
+Let’s talk about communicating internally in Gizra.  
+
+Today, Gizra’s main office is in Tel Aviv Israel (GMT +2) with 16 Israeli employees that use the office in
+varied ways, ranging anywhere from showing up 5 days/week during traditional work hours (~6 people) all the
+way
+to working remotely and coming into the office once/week (~2 people). There is a lot of flexibility both in
+work times and where employees sit and this is mostly determined by role and the person’s circumstances.  
+
+Gizra’s recent hiring strategy has introduced a wonderful group of distributed employees that form almost half
+the company, span across 7 countries (North America and Europe), and several timezones.
+
+This has been wonderful for Gizra but it also comes with the need to rethink how we communicate with each
+other. This is very important and we ALL need to be mindful of it.  
+
+##Tools we have available
+
+###Github
+This is where the project development is communicated.
+The chapters about [Pull Requests] (https://www.thegizraway.com/pull_requests.html) covers how to communicate in Github. However, it is worth mentioning that the
+goal is to write liberally and make things easy to understand so anyone, at anytime, located anywhere can
+read it, and know how to proceed. Note: communicating with clients occurs in the issues, and not in the PR's.
+
+###Slack
+This is where the group (and if you prefer also individual) chats happen. Each channel should have an owner
+to moderate and ensure that questions are answered and help given.
+
+1. “general” channel - a place to say hi, watercooler talk, ask questions that make sense to ask the whole
+company.
+
+2. Role specific groups like “devstuff” or “account_managers”.
+
+3. Project specific groups. It depends on the project but most projects have two groups. The first for team
+discussions, questions, or an ask for help that does not make sense to have in Github.  The second group
+usually named “projectname_github” is Github integrated for anyone who wants Slack notifications for Github
+activity.
+
+4. Other: fun stuff like “nuntius_talks” bot or anything else that someone would like to open.
+
+###Hangouts
+This is where video conversations (and if you prefer also individual chats) happen. Make sure you have a
+working camera, speaker, and earphones. Not everything can be done in asynchronous communication. There are
+times that conversational problem solving is needed. When meeting, turn on your video - visual queues
+are important in conversation.
+
+###The ping (and pong)
+
+Using the chat (Slack or Hangouts) is the equivalent of poking your head into someone’s office to ask a quick question like  “what was that word for…?” It’s that quick answer you need to keep your work moving along.
+
+Initiating conversation with a “ping” can be more effective than if you were sitting
```

Comment:
Are you saying here that all conversations should start with a "ping"? (the actual word typed out). IMO the question itself serves as the ping. Sometime synchronous conversation is not necessary - I could get an answer 5 hours later... still helpful?